Modules/Microsoft365DSC/Dependencies/Manifest.psd1

```diff
@@ -56,6 +56,10 @@
             ModuleName      = 'Microsoft.Graph.Beta.DeviceManagement.Enrollment'
             RequiredVersion = '2.24.0'
         },
+        @{
+            ModuleName      = 'Microsoft.Graph.Beta.NetworkAccess'
+            RequiredVersion = '2.23.0'
```

Comment:
Please update this to be version 2.24.0.

Reply:
updated. Thanks